### ISSUE TITLE
AUR package out of date

### ISSUE THREAD

**Issue by marcusatiliusregulus** [Morganamilo/paru#894]
@Morganamilo
the paru AUR [package](https://aur.archlinux.org/packages/paru) is out of date while paru-bin is not. please update it! thanks

**Comment by z0w13** [comment]
Except (unless I'm misunderstanding) rebuilding the package still leaves it as 1.11.1 as the PKGBUILD is set to download the 1.11.1 source archive

**Comment by nils-werner** [comment]
Wouldn't it make sense to bump [`pkgrel`](https://wiki.archlinux.org/title/PKGBUILD#pkgrel) instead of `pkgver` to trigger a rebuild?

**Comment by mskiptr** [comment]
@nils-werner Indeed, but it's too late for that now. Just bumping `pkgrel` won't bring the AUR version to newest – since 1.11.2 exists.

Still, there's no point in 'updating' from 1.11.1 to 1.11.2 except to make the numbers match.
(It would probably make complaining people stop, tho)

Is cargo or makepkg intelligent enough to skip the build in this case? If not, then it would waste a few joules per person I guess.

**Comment by nils-werner** [comment]
> Still, there's no point in 'updating' from 1.11.1 to 1.11.2 except to make the numbers match.

Of course it makes sense, and exactly for that reason. The packages version number's *sole purpose* is to represent the upstream version number. I doesn't care about what a version number change "means".

> Is cargo or makepkg intelligent enough to skip the build in this case?

Yes, cargo is smart enough. In most cases it will only be a repackage.
